ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -134,3 +134,8 @@ dock_questionSet_enable={{ dock_questionSet_enable | default('true') }}
 
 #release-4.0.0
 sunbird_base_proto={{sunbird_base_proto | default(proto)}}
+
+#release-4.1.0
+opensaber_service_url={{ sunbird_opensaber_service_url }}
```

Comment:
Please add the default value `http://opensaber-service:8080`

Reply:
@keshavprasadms Done. Please check